apps/clearance_ui/src/components/clearance_library/bulk_conclusions/TableCell.tsx

```diff
@@ -0,0 +1,65 @@
+// SPDX-FileCopyrightText: 2024 Double Open Oy
+//
+// SPDX-License-Identifier: MIT
+
+import { useEffect, useState } from "react";
+import { CellContext } from "@tanstack/react-table";
+import { ZodiosResponseByAlias } from "@zodios/core";
+import { userAPI } from "validation-helpers";
+import { Input } from "@/components/ui/input";
+import { Textarea } from "@/components/ui/textarea";
+
+type BulkConclusion = ZodiosResponseByAlias<
+    typeof userAPI,
+    "GetBulkConclusions"
+>["bulkConclusions"][0];
```

Comment:
Nit, I believe this is a little more idiomatic

```suggestion
>["bulkConclusions"][number];
```